app/services/subscriptions/override_service.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+module Subscriptions
+  class OverrideService < BaseService
+    include ChargeModelAttributesHandler
```

Comment:
I think we should not use GraphQL helper at this level. Input arguments should be prepared before calling the service.

For example, the signature of the method could be:
```ruby
def call(paln_args:, subscription_args:)
```

Reply:
refactored